graphql.go

```diff
@@ -45,7 +42,7 @@ type Params struct {
 
 func Do(p Params) *Result {
 	if manager == nil {
-		manager = newResolveManager(defaultWorkers)
+		manager = newResolveManager()
```

Comment:
Speaking of race conditions... what prevents this being done twice?

Reply:
I wrapped that in a sync.Once now